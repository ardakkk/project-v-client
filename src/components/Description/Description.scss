.desc {
    display: flex;
}

.item {
    margin-right: 15px;
}