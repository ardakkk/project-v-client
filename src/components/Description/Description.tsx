import * as React from 'react';
import * as classes from  './Description.scss';
const logo = require('../../assets/images/react.png');
interface DescriptionProps {
}

export const Description: React.FunctionComponent<DescriptionProps> = (props) => {

    return (
        <div className={ classes.desc }>
            <img src={ logo } alt='logo'/>
            <div className={ classes.item }>1</div>
            <div className={ classes.item }>2</div>
            <div className={ classes.item }>3</div>
            <div className={ classes.item }>4</div>
            <div className={ classes.item }>5</div>
        </div>
    );
};