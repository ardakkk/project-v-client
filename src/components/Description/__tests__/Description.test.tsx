import * as React from 'react';
import * as renderer from 'react-test-renderer';

import { Description } from '../Description';

it('renders correctly', () => {
    const tree = renderer
      .create(<Description />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });