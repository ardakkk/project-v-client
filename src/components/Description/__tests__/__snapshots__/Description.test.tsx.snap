// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    alt="logo"
    src="test-file-stub"
  />
  <div>
    1
  </div>
  <div>
    2
  </div>
  <div>
    3
  </div>
  <div>
    4
  </div>
  <div>
    1215
  </div>
</div>
`;
